abi.html

```diff
@@ -5029,6 +5029,12 @@
 		 ::= L &lt;<a href="#mangle.type">type</a>&gt; &lt;<i>real-part</i> <a href="#mangle.float">float</a>&gt; _ &lt;<i>imag-part</i> <a href="#mangle.float">float</a>&gt; E   # complex floating point literal (C 2000)
                  ::= L &lt;<a href="#mangle.mangled-name">mangled-name</a>&gt; E                                 # external name
 
+  &lt;<a name="mangle.braced-expression">braced-expression</a>&gt; ::= &lt;<a href="#mangle.expression">expression</a>&gt;
+                      ::= di &lt;<i>field</i> <a href="#mangle.source-name">source-name</a>&gt; &lt;<a href="#mangle.braced-expression">braced-expression</a>&gt;    # .name = expr
+                      ::= dx &lt;<i>index</i> <a href="#mangle.expression">expression</a>&gt; &lt;<a href="#mangle.braced-expression">braced-expression</a>&gt;     # [expr] = expr
+                      ::= dX &lt;<i>range begin</i> <a href="#mangle.expression">expression</a>&gt; &lt;<i>range end</i> <a href="#mangle.expression">expression</a>&gt; &lt;<a href="#mangle.braced-expression">braced-expression</a>&gt;
+                                                                        # [expr ... expr] = expr
+
```

Comment:
I think the fact the braced-expression production recurses back to itself is somewhat self-explanatory, but I added some extra text to remove any ambiguity regarding the order in which the designators in a designator-list should be mangled.